app/templates/Donation_Cancellation_Confirmation.twig

```diff
@@ -1,9 +1,19 @@
 <div id="greenBox" class="inhalt boxShadow">
 
 	<div class="text">
-		<h2>{$ 'donation-cancellation-header'|trans({}) $}</h2>
+		<h2>
+			{% if cancellationSuccessful %}
+			{$ 'donation-cancellation-header'|trans({}) $}</h2>
```

Comment:
You can do just `{$ 'donation-cancellation-header'|trans $}` when you have no variables.